src/basic/log.c

```diff
@@ -75,6 +76,20 @@ static char *log_abort_msg = NULL;
                 }                                                       \
         } while (false)
 
+static int format_time(char *buffer, int len) {
+        time_t t;
+        struct tm tm;
+
+        t = (time_t) (now(CLOCK_REALTIME) / USEC_PER_SEC);
+        if (!localtime_r(&t, &tm))
+                return -EINVAL;
+
+        if (strftime(buffer, len, "%h %e %T ", &tm) <= 0)
+                return -EINVAL;
+
+        return 0;
```

Comment:
Why not use on of the existing formatting functions in `time-util.h`?

Reply:
yes, please, in particular as this output should closely resemble what "journalctl" would generate